tests/test_common.py

```diff
@@ -32,6 +32,31 @@ def test_stream():
     tables = camelot.read_pdf(filename, flavor="stream")
     assert df.equals(tables[0].df)
 
+def test_password():
+    df = pd.DataFrame(data_stream)
+
+    filename = os.path.join(testdir, "health_protected.pdf")
+
+    try:
```

Comment:
Sorry for so many reviews, I promise this is the last one. A better place for this exception check would be a test inside `test_error.py`. Can you please move it there? (keeping the last two asserts for ownerpass and userpass here)

Reply:
No problem at all, happy to help